lib/normalizers/v1.js

```diff
@@ -335,7 +336,9 @@ module.exports = {
             normalized,
             builders = new Builders(options);
 
-        try { normalized = builders.request(_.cloneDeep(request)); }
+        options && !options.mutate && (request = _.cloneDeep(request));
+
+        try { normalized = builders.request(request); }
```

Comment:
Should the default be mutatable or immutable? I think current version is `immutable`, this might be breaking don't you think?

Reply:
Will add a comment for brevity.